src/stream_compression.jl

```diff
@@ -170,7 +170,7 @@ function TranscodingStreams.finalize(codec::LZ4Decompressor)::Nothing
 end
 
 """
-Deompresses the data from `input` and writes to `output`.
+Decompresses the data from `input` and writes to `output`.
```

Comment:
Not super necessary, but you could add the function signatures to the docstrings for the sake of style.